@here/harp-mapview/lib/Tile.ts

```diff
@@ -685,23 +687,52 @@ export class Tile implements CachedResource {
      * @note Negative values indicates depressions.
      */
     get minElevation(): number {
-        return this.m_minElevation;
-    }
-
-    set minElevation(elevation: number) {
-        this.m_minElevation = elevation;
+        return this.m_minElevation ?? 0;
     }
 
     /**
      * Estimated maximum ground elevation above the sea level that may be found on tile.
      * @note Negative values indicates depressions.
      */
     get maxElevation(): number {
-        return this.m_maxElevation;
+        return this.m_maxElevation ?? 0;
     }
 
-    set maxElevation(elevation: number) {
-        this.m_maxElevation = elevation;
+    /**
+     * @internal
+     * Sets the tile's ground elevation range.
+     *
+     * @param minElevation Minimum tile elevation in meters.
+     * @param maxElevation Maximum tile elevation in meters.
+     */
+    setElevation(minElevation: number, maxElevation: number) {
+        if (this.m_minElevation === minElevation && this.m_maxElevation === maxElevation) {
+            return;
+        }
+        this.m_minElevation = minElevation;
+        this.m_maxElevation = maxElevation;
+
+        if (this.m_maxGeometryHeight !== undefined) {
```

Comment:
Would be nice to extend the `Tile` unit test to also include the new methods that update the bounding box.

Reply:
Will do.

Reply:
I saw it is done, so marking as resolved.